smdebug_rulesconfig/actions/utils.py

```diff
@@ -2,20 +2,15 @@
 import json
 
 
-TRAINING_JOB_PREFIX_REGEX = "[A-Za-z0-9\-]+"
```

Comment:
Can you add a unit test to verify the behavior of these regexes?

Reply:
There are already unit tests that test behavior of regexes.

Reply:
https://github.com/awslabs/sagemaker-debugger-rulesconfig/blob/master/tests/core/actions/test_actions.py

Reply:
Can you link them here?